app/controllers/posts_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   def show
     @post = Post.includes(:comments, comments: [:author]).find_by(id: params[:id])
+    @like = Like.find_by(author: current_user, post: @post)
   end
 
   def new
```

Comment:
- [ ] It's a requirement that we use **CanCanCan** for Post authorization in `Post Controller`, kindly do so

e.g
```
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```